### DISCUSSION TITLE
1 week after launch

### DISCUSSION THREAD

**Discussion by loppefaaret** [Chia-Network/chia-blockchain#1527] [General]
So... here we are a week after the launch of the most promising green crypto.

First off, a big congratulation to Bram & Gene, and all you developers carrying these two on your shoulders!
An absolutely amazing launch - and im sure it was even greater than any of you dared to hope for.
I must say, that had you told me a month ago, that we hit 150pibs today... I would had bet my hat that would only happen in a fairytale.
But here we are, not only with 150pibs - but with a substantial number above 10k+ unique ipv4 addresses. 1700+ unique addresses are holding XCH, and a decentralisation number around 0.8... 
All these numbers are thanks to @freddiecoleman and his amazing [chiaexplorer.com](https://www.chiaexplorer.com/)

**Comment by GeorgePapageorgakis** [comment]
10K ips and only 1200 wallets have 2+ XCH...
500 have 10 or more XCH. Distribution is weird...

**Reply by hoffmang9** [reply]
Are you trying to run a regular Timelord? 10k ips is way behind the current top speeds of 195k ips so it would be a much better use of resources to run a Bluebox. However timelording and farming on the same under powered machine can get in the way of farming as timelord is hard on CPUs.

**Reply by mariano54** [reply]
ip addresses not iterations per second :P
